io.catenax.special_characteristics.measurement/1.0.0/Measurement.ttl

```diff
@@ -0,0 +1,83 @@
+
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.special_characteristics.measurement:1.0.0#> .
+@prefix bpn: <urn:samm:io.catenax.shared.business_partner_number:2.0.0#> .
+@prefix samm-u: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix serialPart: <urn:samm:io.catenax.serial_part:3.0.0#> .
+
+:SpecialCharacteristicMeasurement a samm:Aspect ;
+   samm:preferredName "SpecialCharacteristicMeasurment"@en ;
+   samm:description "Contains the Measurment for a special characteristic"@en ;
+   samm:properties ( serialPart:localIdentifiers bpn:bpnlProperty serialPart:customerPartId :characteristicId :results [ samm:property :measurementType; samm:optional true ] :revisionIndex ) ;
```

Comment:
To match the appropriate _SerialPart_ aspect model, use **manufacturerID** as name to describe the **bpnlProperty**. 

To do this, model a property called “manufactureID” and reference the _bpnlTrait_ from the BusinessPartnerNumber 2.0.0 aspect model, which contains the BPNL characteristics and constrains: https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.shared.business_partner_number/2.0.0

Reply:
done